operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -48,6 +48,14 @@ function check_for_shutdown() {
     return 0
   fi
 
+  if [ -e /tmp/diefast ]; then
```

Comment:
This scares me 😨

Reply:
This can cut several seconds off of shutdown, and we don't need a (slow) clean shutdown for the test cleanup case.  Would you like a different name that's more clear?  `bypassSafeShutdown` or some-such?

Reply:
I'll keep it as is... I just said that it was scary.